### PR TITLE
BookingPool: Fix iterating over non array

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Mantis Issue: https://mantis.ilias.de/view.php?id=31924

### CODE REVIEW

File: Modules/BookingManager/Objects/classes/class.ilBookingObjectsTableGUI.php [hunk 1]
```diff
@@ -342,10 +344,11 @@ protected function fillRow($a_set)
                 $assign_possible = false;
             }
         } elseif (!$this->may_edit) {
-            foreach ($this->reservations[$a_set["booking_object_id"]] as $item) {
-                if ($item["status"] != ilBookingReservation::STATUS_CANCELLED &&
```
Comment (alex40724) [inline Modules/BookingManager/Objects/classes/class.ilBookingObjectsTableGUI.php:346]: Why has this line be removed? The status can be cancelled, right?

**Comment by alex40724** [comment]
@mjansenDatabay Thanks for the PR. I am unsure if the fix is correct, I will need more time to reproduce / test this.

File: Modules/BookingManager/Objects/classes/class.ilBookingObjectsTableGUI.php (hunk 1 above)
Comment (mjansenDatabay) [reply Modules/BookingManager/Objects/classes/class.ilBookingObjectsTableGUI.php:346]: Looks like copy/paste, I'll check this.

Comment (mjansenDatabay) [reply Modules/BookingManager/Objects/classes/class.ilBookingObjectsTableGUI.php:346]: Fixed

**Comment by alex40724** [comment]
Thanks again. This should be fixed by https://github.com/ILIAS-eLearning/ILIAS/commit/6a813f003ada1cc4a692bcd789650547a8d4dcd8

**Pull request closed without merging**